### DISCUSSION TITLE
Keys

### DISCUSSION THREAD

**Discussion by Winbird107** [skyline-emu/skyline#181] [Q&A] [answered]
Alguien que pueda pasar las keys de los titulos yas portadas de los juegos

**Comment by nekomekoraiyuu** [comment, marked as answer]
NO.
Piracy is pretty much prohibited.
Please do dump the keys legally from your switch,
thank you.

**Reply by Douglasradio13** [reply]
E você está em uma discussão de emulação é um baita hipócrita sem noção, vai lá gastar 70 dólares em jogo switch gadozao da Nintendo
